petpal/pipelines/preproc_steps.py

```
@@ -671,5 +674,34 @@ def default_register_pet_to_t1(cls, reference_image_path='', half_life:float=Non
             warnings.warn(f"Invalid override: {err}. Using default instance instead.", stacklevel=2)
             return cls(**defaults)
 
+    @classmethod
+    def default_rescale_image(cls, **overrides):
+        r"""Creates a default step-instance for rescaling an image using :class:`rescale_image<petpal.preproc.image_operations_4d.rescale_image>`.
+
+        The defaults for this step will divide the input image by 37000.0 which is usually done to go
+        from kBq/ml to nCi/ml.
+
+        Notes:
+             The function :class:`rescale_image<petpal.preproc.image_operations_4d.rescale_image>` is wrapped using
+             :func:`ANTsImageToANTsImage<petpal.utils.decorators.ANTsImageToANTsImage>` since steps require input and output
+             paths.
+
+        Args:
+            **overrides:
+
+        Returns:
+            ImageToImageStep: A new step instance for rescaling the input image.
+        """
```

Small thing, but I've noticed a potential drawback to these default functions is that they obscure the name of the step. I think a quick and sufficient solution would just be to explicitly state the default step name somewhere in the docstring so an end user doesn't have to look at the source code or run print_steps_names() later to figure it out. Or maybe better yet, put name as an argument to the function with the default set, just so it's a little more visible.

---

This is a great idea, and I will do that!